src/vm/i386/umthunkstub.S

```diff
@@ -37,6 +37,7 @@ NESTED_ENTRY UMThunkStub, _TEXT, UnhandledExceptionHandlerUnix
 // saved edi
 // UMEntryThunk*
 // Thread*
+// dummy void* for 16 byte stack alignment
```

Comment:
A nit, the comment that you had there before ("dummy 4 bytes" instead of "void*") seemed better to me.